test/integration/Migration.hs

```diff
@@ -1,8 +1,10 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DerivingStrategies #-}
 {-# LANGUAGE ExistentialQuantification #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
```

Comment:
are these still needed?

Reply:
just checked. Yes, they are still needed.

Reply:
oh i see. i'm not running CI on the latest GHC version